bin/monitor.ml

```diff
@@ -163,21 +232,164 @@ end = struct
     | Diagnostics diagnostics -> Update.diagnostics state diagnostics
   ;;
 
+  module Jobs_tab : sig
+    val draw : t -> min_duration_sec:float -> min_display_duration_sec:float -> unit
+  end = struct
+    open Dune_tui.Import
+
+    let render (state : Job_state.t) ~min_duration_sec ~min_display_duration_sec =
```

Comment:
It should be easy enough to keep this outside this module with a little functor over the rendering calls as well.